pkgs/development/tools/jdt-language-server/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, jdk
```

Comment:
let's specify the JDK version here specifically.

Reply:
I'll set it to `jdk15` since that is now in master. This is relevant to your other comment below about whether it needs to be the same version as the user's project. It doesn't, but it does need to be >= to it. So if we had this at jdk 13, then jdk 15 projects wouldn't work.